bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/api/OpenSprinklerHttpApiV210.java

```diff
@@ -58,7 +62,7 @@ public boolean isStationOpen(int station) throws Exception {
                     + " but station " + station + " was requested for a status update.");
         }
         try {
-            returnContent = Http.sendHttpGet(getBaseUrl() + CMD_STATION_INFO, getRequestRequiredOptions());
+            returnContent = http.sendHttpGet(getBaseUrl() + CMD_STATION_INFO, getRequestRequiredOptions());
         } catch (Exception exp) {
```

Comment:
I know it's not your fault, but can you have a look at the try/catch-blocks here?

Reply:
Absolutely!